Rakefile

```diff
@@ -23,7 +33,116 @@ task :travis do
   Rake::Task['spec'].invoke
 end
 
+module HarnessOptions
+  defaults = {
+      :tests  => ['tests'],
+      :log_level => 'debug',
+      :preserve_hosts => 'onfail',
+  }
+
+  DEFAULTS = defaults
+
+  def self.get_options(file_path)
+    puts "Attempting to merge config file: #{file_path}"
+    if File.exists? file_path
+      options = eval(File.read(file_path), binding)
+    else
+      puts "No options file found at #{File.expand_path(file_path)}... skipping"
+    end
+    options || {}
+  end
+
+  def self.get_local_options
+    get_options("./local_options.rb")
```

Comment:
This is actually relative to the rakefile itself, so it shouldn't matter where rake is called from. I've tested this by running rake from different directories. The local_options.rb should be at the same level as the Rakefile. I will update the task usage to make it clearer.